client/keystore/src/local.rs

```diff
@@ -301,6 +310,19 @@ impl SyncCryptoStore for LocalKeystore {
 			Ok(None)
 		}
 	}
+
+	fn ecdsa_sign_prehashed(
+		&self,
+		id: KeyTypeId,
+		public: &ecdsa::Public,
+		msg: &[u8; 32],
+	) -> std::result::Result<Option<Vec<u8>>, TraitError> {
+		let pair = self.0.read()
+			.key_pair_by_type::<ecdsa::Pair>(public, id)
+			.map_err(|e| TraitError::from(e))?;
```

Comment:
```suggestion
			.key_pair_by_type::<ecdsa::Pair>(public, id)?;
```
That should suffice?